policies/ecs-task-execution-role-policy.json

```diff
@@ -1,17 +0,0 @@
-{
```

Comment:
As you may have seen, all policies that are used in this module, are stored in the `policies` directory. Let's keep that design 🙏.

To include a computed value into the policy file, we can reference it by using the `template` provider.

Eg. https://github.com/strvcom/terraform-aws-fargate/blob/master/main.tf#L91-L99